__tests__/sources.ts

```diff
@@ -432,6 +433,27 @@ query SourceMembers($id: ID!) {
     expect(res.data).toMatchSnapshot();
   });
 
+  it('should return source members and order by their role', async () => {
```

Comment:
Can you make sure in this test that the following order holds: Owner, Moderator, Member

Reply:
Oh, right. I did not include the `Member` on the test. Let me add it quickly.